npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser/lockfile_parser.rb

```diff
@@ -188,7 +190,11 @@ def semver_version_for(version_string)
         end
 
         def alias_package?(requirement)
-          requirement.include?("@npm:")
+          Experiments.enabled?(:yarn_berry) ? requirement.match?(/@npm:(.+@(?!npm))/) : requirement.include?("@npm:")
```

Comment:
This is really a matter of personal preference, but I find this line a little harder to parse than normal due to the Regexp syntax combining with the ternary syntax. What do you think of 👇?

```suggestion
          if Experiments.enabled?(:yarn_berry)
            requirement.match?(/@npm:(.+@(?!npm))/)
          else  
            requirement.include?("@npm:")
          end
```